pkg/restapi/operation/operations_test.go

```diff
@@ -435,7 +453,7 @@ func TestQueryVault(t *testing.T) {
 		require.NoError(t, err)
 
 		urlVars := make(map[string]string)
-		urlVars[vaultIDPathVariable] = testVaultID
+		urlVars[vaultIDPathVariable] = testReferenceID
```

Comment:
While it technically doesn't matter for this test, it's a little odd to use a reference ID as a vault ID. Sometimes people look at the tests to understand how everything is supposed to work, so it's good to try to use reasonably valid examples where possible.

Reply:
fixed